scanner/services/matcher.go

```diff
@@ -16,18 +23,83 @@ import (
 // matcherService represents a vulnerability matcher gRPC service.
 type matcherService struct {
```

Comment:
Goland doesn't seem to like us returning an unexported return type. Maybe we should export this

Reply:
It's fine like this.